app/resources/views/pages/location_management/suggestion_list.blade.php

```diff
@@ -0,0 +1,17 @@
+@extends('layouts.default')
+@section('content')
+<div class="suggestion-list">
+	<div class="text-center">
+		<h1><a href="/location/management/my-locations">Suggesstions for {{ $name }}</a></h1>
+	</div>
+	<div class="list-group">
+		@foreach($suggestions as $suggestion)
+		<a href="/suggestion-detail/{{ $suggestion->id }}" class="list-group-item" title="Click to check details">
+			<span class="username">Generated by: {{ $suggestion->user_name }}</span>
```

Comment:
Instead of "Generated", use "Submitted".  "Generated" sounds more like it was made by a machine.